### PR TITLE
Smaller post header images

### LINKED ISSUE #20: Home modifications
Let's change the home preview so that this doesn't happen.

A user entering the website should see at least a couple of articles on first sight.

image

### PR DESCRIPTION
**Pull request by dcgoodridge**
Closes #20

### CODE REVIEW

File: _layouts/page.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
                     <div class="entry-header-wrap">  
                         {% if page.banner_image %}
                         <div class="post-thumbnail">
-                            <img src="{{ "/images/" | append: page.banner_image | prepend: site.baseurl }}" alt="{% if page.banner_image_alt %}{{ page.banner_image_alt }}{% else %}{{ page.title }}{% endif %}">
+                            <img class="post-thumbnail" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=" style="background-image: url({{ "/images/" | append: page.banner_image | prepend: site.baseurl }});" alt="{% if page.banner_image_alt %}{{ page.banner_image_alt }}{% else %}{{ page.title }}{% endif %}">
```
Comment (DannieBGoode) [inline _layouts/page.html]: what is this src?

File: css/main.scss [hunk 2]
```diff
@@ -791,6 +795,11 @@ a.archive-top-link:hover {
         margin-right: auto;
         max-width: 500px;
     }
+	
+	.post-thumbnail {
+		height: 360px;
```
Comment (DannieBGoode) [inline css/main.scss]: duplicated code, delete this

File: css/main.scss [hunk 3]
```diff
@@ -888,7 +897,12 @@ a.archive-top-link:hover {
 
     #menu-toggle.toggled-on:before {
         content: "\f00d";
-    }    
+    } 
+
+	.post-thumbnail {
```
Comment (DannieBGoode) [inline css/main.scss]: duplicated code

File: _layouts/page.html [hunk 4]
```diff
@@ -9,7 +9,7 @@
                     <div class="entry-header-wrap">  
                         {% if page.banner_image %}
                         <div class="post-thumbnail">
```
Comment (DannieBGoode) [inline _layouts/page.html]: I dont like the fact that the parent and the child have the same class "post-thumbnail", we should use proper css selectors.

File: _layouts/page.html (hunk 1 above)
Comment (dcgoodridge) [reply _layouts/page.html]: 1x1 transparent pixel
generated at http://png-pixel.com/

**Pull request merged**